### PR TITLE
creating tables, pie and bar plots

### PR DESCRIPTION
**Pull request by Phoenix795**
version one

### CODE REVIEW

File: plots.py [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+import pandas as pd
+import reading_database as rd
+import os
+
+PATH = 'figs'
+
+def write_html_to_file(filename, dataframe):
+    """Writing dataframe to HTML file"""
+    html = dataframe.to_html(index=False)
+    text_file = open(f'{PATH}/{filename}.html', "w")
```
Comment (Roxe322) [inline plots.py]: with

File: plots.py [hunk 2]
```diff
@@ -0,0 +1,118 @@
...
+        )
+    plt.title("Distribution of completed issues", fontsize=25)
+    plt.xlabel('Version', fontsize=20)
+    plt.ylabel('Number of Issues', fontsize=20)
+    bar_diagram.figure.savefig(f'{PATH}/completed_issue_destribution')
+
+
+if __name__ == "__main__":
+    os.makedirs(PATH,exist_ok=True)
+    most_voted_suggestions_table()
```
Comment (Roxe322) [inline plots.py]: Лучше поменять на цикл

File: plots.py [hunk 3]
```diff
@@ -0,0 +1,118 @@
...
+    most_voted_bugs_table()
```
Comment (Roxe322) [inline plots.py]: functions_to_call = [most_voted_suggestions_table, most_voted_bugs_table, ...]
for func in functions_to_call:
     func()

File: plots.py (hunk 1 above)
Comment (Phoenix795) [reply plots.py]: done

File: plots.py (hunk 2 above)
Comment (Phoenix795) [reply plots.py]: done

File: plots.py (hunk 3 above)
Comment (Phoenix795) [reply plots.py]: Done

**Pull request merged**